test/fixtures/wsl_products.yml

```
@@ -0,0 +1,12 @@
+# TODO: references for this file,
+# describe what it is
+# https://jira.suse.com/browse/PED-1380
+- display_name: "SUSE Linux Enterprise Desktop 15 SP4"
+  name: "SLED"
+  register_target: "sle-15-$arch"
+  version: 15.4
```

Add quotes, otherwise the value is loaded as `Float` type, that might have some nasty side effects...
```suggestion
  version: "15.4"
```

```
> irb -r yaml
irb(main):001:0> YAML.load_file("test/fixtures/wsl_products.yml").first["version"]
=> 15.4
irb(main):002:0> YAML.load_file("test/fixtures/wsl_products.yml").first["version"].class
=> Float
```

---

it is intentional. We convert it to string in parser.